server/config/config.go

```diff
@@ -490,6 +490,11 @@ type ScheduleConfig struct {
 	LeaderScheduleLimit uint64 `toml:"leader-schedule-limit,omitempty" json:"leader-schedule-limit"`
 	// LeaderScheduleStrategy is the option to balance leader, there are some strategics supported: ["count", "size"], default: "count"
 	LeaderScheduleStrategy string `toml:"leader-schedule-strategy,omitempty" json:"leader-schedule-strategy,string"`
+	// FlexibleScore is the option to set max score.
+	// When start cluster, get the maximum capacity in the cluster, and then let max_score = max_capacity + flexible_score.
+	// When add new store, if capacity of the new store is lower than max_score, max_score will not be adjusted.
+	// Otherwise, we will set the max_score of all the store to the new one.
+	FlexibleScore uint64 `toml:"flexible-score,omitempty" json:"flexible-score"`
```

Comment:
I think we need a better name, but I have no idea.

Reply:
PaddingScore?

Reply:
Besides, we have too many configuration options...

Reply:
> Besides, we have too many configuration options... It's a burden for users.

Good, you are right. I am testing to ensuring a default value. It can avoid user to modify the option.